GitUI/Design/ExecutableFileNameEditor.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Windows.Forms;
+using System.Windows.Forms.Design;
+
+namespace GitUI.Design
+{
+    internal class ExecutableFileNameEditor : FileNameEditor
+    {
+        protected override void InitializeDialog(OpenFileDialog openFileDialog)
+        {
+            base.InitializeDialog(openFileDialog);
+            openFileDialog.Filter = "Executable file (*.exe;*.cmd;*.bat)|*.exe;*.cmd;*.bat|All files (*.*)|*.*";
```

Comment:
+ "*.ps1"?

Reply:
I considered that, but PowerShell scripts are executed by PowerShell interpreter, that must be invoked explicitly. E.g. `> powershell.exe .\somescript.ps1`